### ISSUE TITLE
copy/remove failed when shellslash is set under Windows

### ISSUE THREAD

**Issue by ZSaberLv0** [scrooloose/nerdtree#776]
_To assist in resolving your issue, provide as much information as possible, in place of the ellipses (`…`) below._

---
**Environment:** _Describe your Vim/NERDTree setup._

>* Operating System: Windows 7
>* Vim version `:version`: vim 8.0
>* NERDTree version `git rev-parse --short HEAD`: b89de09
>* NERDTree settings applied in your vimrc, if any:
>
>   ```
>   set shellslash
>   ```

**Process:** _List the steps that will recreate the issue._

navigate to nerdtree and press `m` to popup menu, press `c` or `d` to copy or delete node, then it would fail, while `m` to move the node would success

`set noshellslash` before popup the menu would solve the issue

however, simply `unset>save>restore` `shellslash` in `lib/nerdtree/path.vim`'s `s:Path.copy(dest)` and `s:Path.delete()` doesn't fix this problem

**Comment by PhilRunninger** [comment]
I just merged #935, and `m`-`d` is working now. `m`-`c` still fails, and I will take a look at that next.

**Comment by ZSaberLv0** [comment]
@PhilRunninger #935 did solves `m-c`, but I think you forgot `m-d`, could you please fix that too?

PS: I've no idea why changing `s:Path.copy(dest)` / `s:Path.delete()` does not solve the issue

**Comment by PhilRunninger** [comment]
I just pushed another commit to fix `m-d`. I didn't catch it before, because it seems to have been broken only if `g:NERDTreeRemoveFileCmd` exists; otherwise, Vim's internal `delete()` function is used.

The reason I didn't modify `path.vim` is because `s:Path.copy()` would receive the destination with forward slashes, and then append it to the copy command. `copy c:/tmp/foo c:/tmp/bar` then fails when it's sent to Windows. I chose to modify `fs_menu.vim` to ensure the path would be sent to `s:Path.copy()` in the correct format.
